config/nginx.conf

```diff
@@ -31,5 +31,8 @@ http {
     server {
         listen 3000;
         listen [::]:3000;
+        location / {
+            root /usr/share/nginx/html;
+        }
```

Comment:
This can be removed. We reverted back to node.js demo example in all stacks so nginx should be removed.